draft-ietf-taps-interface.md

```diff
@@ -440,6 +443,7 @@ C.Send(messageDataRequest)
 // Only receive complete messages
 C.Receive()
 //---- Ready event handler for any Connection C end ----
+messageContext := NewMessageContext()
```

Comment:
This part did not change so why is this now needed to create the messageContext?

Reply:
```suggestion
```